src/homework5/task1.py

```diff
@@ -0,0 +1,25 @@
+# Оформите решение задач из прошлых домашних работ в функции.
+# Напишите функцию runner. (все станет проще когда мы изучим модули, getattr и setattr)
+# runner() – все фукнции вызываются по очереди
+# runner(‘func_name’) – вызывается только функцию func_name.
+# runner(‘func’, ‘func1’...) - вызывает все переданные функции
+from inspect import isfunction
+import task1_func
+
+
+def runner(*args):
+
+    list_of_attr = dir(task1_func)
+    for func in list_of_attr:
+        attr = getattr(task1_func, func)
```

Comment:
what about magic methods? they would alco be called in case you haven't received args